perun-core/src/test/java/cz/metacentrum/perun/core/entry/VosManagerEntryIntegrationTest.java

```diff
@@ -558,6 +559,26 @@ public void getBansForVo() throws Exception {
 		assertThat(voBans).containsOnly(ban);
 	}
 
+	@Test
+	public void updateBan() throws Exception {
+		System.out.println(CLASS_NAME + "updateBan");
+		Vo vo = vosManagerEntry.createVo(sess, myVo);
+		Member member = createMemberFromExtSource(vo);
+
+		BanOnVo banOnVo = new BanOnVo();
+		banOnVo.setMemberId(member.getId());
+		banOnVo.setVoId(vo.getId());
```

Comment:
Technically, you don't have to set VO's ID here because it is set later in the setBan method using member's voId value. However, it's just a tiny detail.